Add health_check django command (#217) (#234)
codingjoe/django-health-check: README.rst

        url(r'^ht/$', views.HealthCheckCustomView.as_view(), name='health_check_custom'),
    ]

Django command
--------------

> You can run django command `health_check` to perform command line checks in a shell,

Reviewer: ```suggestion
You can run the Django command `health_check` to perform your health checks via the command line,
```

or periodically with a cron, as follow:

.. code::

    django-admin health_check

> Then you should get something like follow as output:

Reviewer: ```suggestion
This should yield the following output:
```

.. code::

    DatabaseHealthCheck      ... working
    CustomHealthCheck        ... unavailable: Something went wrong!

> In case of critical error (as with http version) the exit status will be `1`.

Reviewer: ```suggestion
Similar to the http version, a critical error will case the command to quit with the exit code `1`.
```
